metric-schema-api/src/main/conjure/metric-schema-api.yml

```diff
@@ -35,6 +35,7 @@ types:
       Documentation:
         docs: Documentation describing an associated component. Markdown syntax may be used.
         alias: string
-      TagValue:
+      TagDefinition:
         fields:
-          value: string
+          name: string
+          values: set<string>
```

Comment:
The `TagValue` type grated us some additional flexibility. Any reason not to include both, using `set<TagValue>` here?